src/Sql/dbo/Stored Procedures/Provider_Create.sql

```diff
@@ -1,54 +0,0 @@
-﻿CREATE PROCEDURE [dbo].[Provider_Create]
```

Comment:
I suspect this was accidentally deleted?

Reply:
ooh, yes. This was an accident. 😳 Thank you!